raw-spans-grouper/helm/values.yaml

```diff
@@ -156,6 +156,12 @@ kafka-topic-creator:
           retention.bytes: 8589934592 # default = -1
           retention.ms: 86400000 # default = 604800000 (7 days)
           max.message.bytes: 10485760 # default = 1048588
+      raw-spans-to-structured-traces-grouping-job-span-data-store-changelog:
+        configs:
```

Comment:
@pavan-traceable without `replicationFactor` and `partitions` configs, this will cause the kafka-topic-creator to fail.

Reply:
What is the default value for partitions ?